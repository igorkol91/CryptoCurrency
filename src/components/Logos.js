import React from 'react';
import { PropTypes } from 'prop-types';

const Logos = ({ name, logoUrl }) => (
  <div className="link-div">
    <img src={logoUrl} alt="img" className="home-links-img" />
    <h3 className="home-links-text">{name}</h3>
  </div>
);

Logos.propTypes = {
  name: PropTypes.string.isRequired,
  logoUrl: PropTypes.string.isRequired,
};

export default Logos;
