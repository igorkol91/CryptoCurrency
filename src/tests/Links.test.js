import renderer from 'react-test-renderer';
import Logos from '../components/Logos';

describe('Logos component', () => {
  test('Snapshot test', () => {
    const tree = renderer.create(
      <Logos />,
    )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
