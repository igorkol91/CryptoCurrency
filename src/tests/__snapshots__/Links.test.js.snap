// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logos component Snapshot test 1`] = `
<div
  className="link-div d-flex align-items-center h-100"
>
  <div
    className="mx-1"
  >
    <h5
      className="home-links-text"
    />
    <h6
      className="home-links-text"
    >
      Rank:
    </h6>
  </div>
</div>
`;
